.env.example

```diff
@@ -69,6 +69,7 @@ SENDGRID_API_KEY=sendgrid-key
 SENDGRID_SENDER_EMAIL=info@podkrepi.bg
 SENDGRID_INTERNAL_EMAIL=dev@podkrepi.bg
 SENDGRID_CONTACTS_URL=/v3/marketing/contacts
+MARKETING_LIST_ID=6add1a52-f74e-4c14-af56-ec7e1d2318f0
```

Comment:
is this the main newsletter list Id for all news from Podkrepi.bg? 
If so then let's call it PODKREPI_NEWSLETTER_LIST_ID